improver/calibration/ensemble_calibration.py

```diff
@@ -37,7 +37,10 @@
    ensemble_calibration.rst
 
 """
+import importlib
+import os
 import warnings
+from multiprocessing import Pool
```

Comment:
I've removed this after speaking to Tomek to understand how this would conflict with existing forms of parallelisation and after checking that the estimate-emos-coefficients CLI will still run in a reasonable time even with the parallelisation removed.